spec/requests/users_spec.rb

```diff
@@ -48,15 +48,23 @@
   end
 
   describe "PATCH /update" do
-    let(:user) { FactoryBot.create(:user) }
+    let(:user) { FactoryBot.create(:user, role: "unset") }
 
     it "updates the user" do
-      skip "something weird is up with this action/test -- @user.update returns true but the email doesn't change"
-
       patch "/users/#{user.id}", params: { user: { email: "atorvingen@example.com" } }
-      user.reload
+
+      # Manually confirming because it's required by Devise when changing a user's email address
+      user.reload.confirm
```

Comment:
It was a lot easier to test an email change than other attributes since they're located on the `Person` model